tdvt/tdvt/config_gen/datasource_list.py

```diff
@@ -435,3 +434,4 @@ class LinuxRegistry(TestRegistry):
 
     def __init__(self):
         super(LinuxRegistry, self).__init__('linux')
+        
```

Comment:
You might need to check your line endings for some of these files.

Reply:
I'll remove the spacing. I had added this since the pep8 bot recommended adding spacing for EOF.